app/src/main/java/com/stevesoltys/seedvault/transport/backup/BackupCoordinator.kt

```diff
@@ -120,18 +116,15 @@ internal class BackupCoordinator(
     suspend fun initializeDevice(): Int = try {
         // we don't respect the intended system behavior here by always starting a new [RestoreSet]
         // instead of simply deleting the current one
-        val token = startNewRestoreSet()
+        startNewRestoreSet()
         Log.i(TAG, "Initialize Device!")
         plugin.initializeDevice()
-        Log.d(TAG, "Resetting backup metadata for token $token...")
-        plugin.getMetadataOutputStream(token).use {
-            metadataManager.onDeviceInitialization(token, it)
-        }
+        startNewRestoreSet()
```

Comment:
We are starting two restore sets, intentionally I suppose? Maybe a comment on why?

Reply:
No, I think this is an oversight on my part. I'll take a look at that separately, as this is more delicate.

Reply:
Yeah, so this wasn't correct. Well spotted. Now, we are only doing this once. I tested it and it should work.